docs/pages/older-versions.mdx

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I think we should forward port this page to `master` and backport it to `branch/v8` and `branch/v7` as well so users of all supported versions can view this page.

Also, just double-checking whether the intention is to make this page accessible only to visitors who know the URL, or if there will be a link from a layout in `gravitational/docs`.

Reply:
Once I get the necessary approvals for this test page it will be included in `master`, but for now I don't want to have to update multiple PRs.  It does not need to be backported - that will be handled during redirection.
Many roads and regex matches will lead here eventually, and there will be a link from the versions dropdown.